python/src/HMatrix.i

```diff
@@ -40,6 +41,7 @@ public:
 
   virtual void compute(OT::UnsignedInteger i, OT::UnsignedInteger j, OT::Matrix* localValues) const
   {
+    OT::InterpreterLock lock;
```

Comment:
Unfortunately this does not fix the current issue we have with parallel hmat
I have to see later

Reply:
Ok, I'll leave it for later then